src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -2,7 +2,7 @@ import { LightningElement, api, track, wire } from 'lwc';
 import CURRENCY from '@salesforce/i18n/currency';
 import { registerListener } from 'c/pubsubNoPageRef';
 import { isNull, showToast, constructErrorMessage, format } from 'c/utilCommon';
-import { HTTP_CODES } from 'c/geConstants';
```

Comment:
@npsp-reedestockton I made this change to move this common constant `PAYMENT_METHOD_CREDIT_CARD` to `geConstants` since both teams use it, what do you think?

Reply:
For my 2 cents, I think this makes sense as long as they are both referencing the same value - which for Gift Entry, is the value of the picklist option on the Payment npe01__Payment_Method__c field (I think). However, if they are pointing to different fields it might be nice to use different constants just in case the values  drift in the future.